docs/user-guide/src/bmo/inspect_annotation.md

```diff
@@ -0,0 +1,39 @@
+# Inspect annotation
+
+The inspect annotation can be used to request the baremetal operator to (re-)inspect a `Ready` BareMetalHost.
+This is useful in case there were hardware changes for example.
+Note that it is only possible to do this when BareMetalHost is in `Ready` state.
+If an inspection request is made while BareMetalHost is any other state than Ready, the request will be ignored.
+
```

Comment:
IMO the below deserves "How to use inspect annotaion" header?

Reply:
Sure I can add it, but we don't have any sub-sections on the other pages, like reboot annotation :thinking:

Reply:
Good point, it is mixed actually, some don't have it and some does (secure boot, live ISO). Let's maybe align all of those in a follow-up PR as discussed offline.